far/FarEng.hlf.m4

```diff
@@ -3208,6 +3208,14 @@ $ #Editor: search/replace#
  Treat input as Perl regular expression (~search~@RegExp@ and ~replace~@RegExpRepl@).
 Each line is processed individually, so multi-line expressions and line break characters will not be found.
 
+ #Fuzzy search#
+ The search will be diacritical insensitive (for example, #ёлочка# will be found in #белочка#),
```

Comment:
Nice example, but it probably won't be obvious to the target audience that the words "spruce" and "squirrel" are unrelated and the search is false positive in this context.
Maybe something like "ape"/"canapé", "he"/"cliché" [etc.](https://en-academic.com/dic.nsf/enwiki/3894487)?

Reply:
Well, I did not mean to demonstrate a false positive case, just wanted something a marginally interesting in the example. I decided not to be a smart aleck and opted for "déjà vu" instead.

Reply:
I mean it is perfectly fine in FarRus.hlf, but probably less interesting in other languages.
Anyway, `déjà vu` is a great example too and probably universally understandable.